client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/sidebar.tsx

```diff
@@ -44,16 +45,24 @@ function getChecklistCompletionProgress( tasks: Task[] | null ) {
 const Sidebar = ( { siteSlug, submit, goNext, goToStep }: SidebarProps ) => {
 	let siteName = '';
 	let topLevelDomain = '';
+	const [ showLaunchTitle, setShowLaunchTitle ] = useState< boolean >( false );
 	const flow = useFlowParam();
 	const translate = useTranslate();
 	const site = useSite();
-	const translatedStrings = getLaunchpadTranslations( flow );
+	const { flowName, title, launchTitle, subtitle } = getLaunchpadTranslations( flow );
 	const arrayOfFilteredTasks: Task[] | null = getArrayOfFilteredTasks( tasks, flow );
 	const enhancedTasks =
 		site && getEnhancedTasks( arrayOfFilteredTasks, siteSlug, site, submit, goToStep );
 
 	const taskCompletionProgress = site && getChecklistCompletionProgress( enhancedTasks );
 
+	useEffect( () => {
+		const launchTask = enhancedTasks?.filter( ( task ) => task.id.includes( 'launched' ) );
```

Comment:
I think it would be more maintainable to turn this into an explicit property on the task object, rather than parse the id for specific terms

Reply:
@jeyip - Sure, I'm fine with that. I was trying to derive the value rather than keep adding properties. But filtering for 'launched' in the task.id is admittedly potentially brittle. I'll update based on this suggestion.

Reply:
@jeyip - This is done. Also added small additional test to check for this property.